src/ui/sass/modules/_Results.scss

```diff
@@ -164,8 +168,6 @@ $results-cards-margin: calc(var(--yxt-base-spacing) / 2) !default;
   &-title svg, &-title img 
   {
     color: var(--yxt-color-brand-primary);
-    height: var(--yxt-base-spacing);
```

Comment:
Just a note: as we confirmed offline, this height and width default to 1em x 1em in the Icon.scss for `Icon svg` and `Icon-image`. Thus we do not need to override it here.